src/http_proxy/Cargo.toml

```diff
@@ -20,10 +20,10 @@ clap = { version = "3.0.0", features = ["derive"] }
 hex = "0.4.3"
 log-panics = { version = "2", features = ["with-backtrace"]}
 minicbor = { version = "0.18.0", features = ["derive", "std"] }
-many-client = { path = "../many-client", version = "0.1.0" }
-many-identity = { path = "../many-identity", version = "0.1.0" }
-many-identity-dsa = { path = "../many-identity-dsa", version = "0.1.0" }
-many-modules = { path = "../many-modules", version = "0.1.0" }
+many-client = { path = "../many-client" }
+many-identity = { path = "../many-identity" }
```

Comment:
I thought I could eliminate the `version` field because of `path`, but your comment made me realize I can't do that.

> Note: [crates.io](https://crates.io/) does not allow packages to be published with git dependencies (git [dev-dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies) are ignored). See the [Multiple locations](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations) section for a fallback alternative.

I will re-add those and ensure they are correctly handled by the `release.sh` script.

Reply:
Done